### ISSUE TITLE
Cannot use py.test as the testrunner

### ISSUE THREAD

**Issue by Themanwithoutaplan** [Pylons/substanced#268]
NB this is not an attempt to shoehorn pytest in.
I use `py.test -xrf --pdb path/to/test` a lot when testing. pytest normally has no problems running unittests but I've come across one faiilure:

```
self = <substanced.folder.tests.test_views.Test_makeob testMethod=test_it>

    def test_it(self):
        from substanced.folder.views import _makeob
        request = testing.DummyRequest()
        request.registry.content = DummyContent()
        value = _makeob(request, None, None, None)
>       self.assertEqual(value, None)
E       AssertionError: 'FILE' != None
```
Would be nice if this could be resolved.

**Comment by Themanwithoutaplan** [comment]
Interesting. It looks like the monkey patching in `Test_multi_upload_submit` is not being cleaned up when pytest runs. Shouldn't the monkey patching of substanced.folder.views,_makeob be handled by setup and teardown?

**Comment by Themanwithoutaplan** [comment]
Fixed by #272
